### DISCUSSION TITLE
Permissions on deeper pages, not only on collections

### DISCUSSION THREAD

**Discussion by michalkos** [outline/outline#3260] [Ideas]
I would like to select permissions on specific pages (with subpages inheriting permissions). Currently it's only possible on Collection level.

**Comment by tommoor** [comment]
We don't have any current plans to implement this functionality, do you have a usecase in mind?

**Reply by napest** [reply]
+1 for this feature. For example, we develop several products, each product is a collection, but we are planning to put here info like strategy, datasets, assesses and etc. which need a stronger data protection. otherwise we have to create several collections for 1 product, which is inconvenient. Another example: if I want to create a private doc for me and a few colleagues, I have to create a collection for that, which is also unstructured
